### ISSUE TITLE
Can't turn on WH517S from integration

### ISSUE THREAD

**Issue by cadetsguy** [JeffSteinbok/hass-dreo#233]
Good Morning,

I've got a WH517S (DR-HSH017S) heater. The integration seems to be able to control the heater but only after it is manually turned on (via remote or button push on the unit itself). I had a different heater that I was able to automate it turning on in several circumstances and would like to replicate that. Is this a safety limitation or is something else going on with my configuration? Otherwise love the built in functionality of the integration. Appreciate any insight. Let me know if logs/diagnostics would be helpful to include. Thanks!

**Comment by cadetsguy** [comment]
Thanks for the extra information ulepr, I was still messing around troubleshooting but I can confirm that I'm experiencing the exact same thing that you are. Hopefully there's a way the developers can change the functionality of the power action in the integration so that it can be used repeatedly. Thanks all!

**Comment by JeffSteinbok** [comment]
Duplicate of #222 I think. Fix forthcoming.
